backend/game/engine/dice_msg_translator.py

```diff
@@ -0,0 +1,22 @@
+def decode_selected_dice_indexes(payload):
+    i = 0
+    indexes = []
+    for index in payload:
+        if index[1]:
+            indexes.append(i)
+        i += 1
+    return indexes
+
+
+def dice_values_message_create(values):
+    msg = []
+    for val in values:
+        translated_val = translate_enum_to_frontend_val(val.name)
+        msg.append([translated_val, False])
```

Comment:
Why is this being sent with false as the 2nd index?

Reply:
This is a front-end compatibility thing. I should add that to a description. Basically it sets front-end to be "not selected" for the re-roll UI.

Reply:
Added method explanation